configs/analyzer.go

```diff
@@ -16,14 +16,14 @@ type Analyzer struct {
 
 func (a Analyzer) Validate() (bool, error) {
 	if a.LogFilesDirectory == "" {
-		return false, errors.New("❕ log files directory was empty")
+		return false, errors.New("❕ 'log files directory' was not set in configuration")
 	}
 	if strings.Contains(a.LogFilesDirectory, "../") {
```

Comment:
I am just curious what is the reason for this check? I understand it is not the best way how to specify the path to log files but erroring seems kinda harsh to me :p

Reply:
It's a very good question.
I believe it is actually redundant. Thank you for noticing. I will update